dotCMS/src/main/java/com/dotmarketing/business/CacheLocator.java

```diff
@@ -399,7 +404,7 @@ public static Cachable getCache (String value) {
 	RulesCache("Rules Cache"),
 	SiteVisitCache("Rules Engine - Site Visits"),
 	NewNotification("NewNotification Cache"),
-
+	VanityURLCache("Vanity URL Cache"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)